airbyte-cdk/python/docs/concepts/full-refresh-stream.md

```diff
@@ -0,0 +1,31 @@
+# The Basic Full-Refresh Stream
+
+Just like any general HTTP request, the basic `HTTPStream` requires a url to perform the request, and instructions
```

Comment:
same comment about moving HTTP stream things out. I am also happy to do this after you merge this PR.